src/spyglass/position/v1/position_dlc_project.py

```diff
@@ -375,9 +378,84 @@ def insert_new_project(
             config_path = config_path.as_posix()
         return {"project_name": project_name, "config_path": config_path}
 
+    @classmethod
+    def add_video_files(
+        cls,
+        video_list,
+        config_path=None,
+        key=None,
+        output_path: str = os.getenv("DLC_VIDEO_PATH"),
+        add_new=False,
+        add_to_files=True,
+        **kwargs,
+    ):
+        if add_new & (not config_path):
+            if not key:
+                raise ValueError(
+                    "at least one of config_path or key have to be passed if add_new=True"
+                )
+            else:
+                config_path = (cls & key).fetch1("config_path")
+        if (not key) & add_to_files:
+            if config_path:
+                if len(cls & {"config_path": config_path}) == 1:
+                    pass
+                else:
+                    raise ValueError(
+                        "Cannot set add_to_files=True without passing key"
+                    )
+            else:
+                raise ValueError(
+                    "Cannot set add_to_files=True without passing key"
+                )
+
+        if all(isinstance(n, Dict) for n in video_list):
```

Comment:
I think there are actually a lot of things that can be cleaned up/refactored to take advantage of Path functionality. Let's hold off for now to make sure we have a working version and then when we refactor Position/have more tests this can be part of that.